packages/suite/src/constants/suite/b64images.ts

```diff
@@ -2,160 +2,111 @@
+    'dD/n7sYCs+5luHV4vr2727KOR9Xpwy1o0PpRd/gV2e/ZJG9PHA81dodc9riowaxflui0q29Uy955\n' +
+    'U95VfvSIipC+9wirR4PLW6YP3zQT6zt9gKd51EXRdEmItNn3J9Nuxsa99tprePDggR4nPZevozi9\n' +
+    'ITs2/sOng5XkcSYlGMukFDGWbdECc0OaiGIJN5JP3v6i8OtoU08SBIG0POnO1CzKhQ6Sw0UDtEBW\n' +
+    'SyTTMvxaIhUJ7EZBCwcwLRPtnR5ZrK/rahZg7p1n7m4t+xsbs2YC1TvfEO6ywKCsiSOkHIJA5N7X\n' +
+    'z2T5MdIMljYL7JoLVGgWfYbMWbi3fV/DijAXEpeLuwqRXhi0/tXjrFmaWBfzmtk6+traF0GPS5l+\n' +
+    'XXcKBpEDuD0ML0R6zsdIP0+c+7nv5xwtmlCW32B25krL+bTmU/25TdvOqM7QgAVzbvq0DjWBigZU\n' +
+    'NllwDTh6k36xnl+lTJDyFMm2yH+3JNuyJa/bl+9/KdmWgQQMycOH+xOzLdrUc79zSN+LJGqGkfzo\n' +
+    'cieQJh+XtxKYgdtr8lJ/TLD/oIPggGRank0YtCwBU6EPYFdZzbzzxONIWNdxSYnrHUF/xu9DlHMY\n' +
+    'fjq6zWrrtg0C80LYVUxF6QXxBlaLBmcuTWWZEOk5bDtniq2iQ1r1wugSrGRCVP9ZmmbVxJITzvjd\n' +
+    'kRm/20Z6gzFrnXEasOhxnFeHRnDLeOmoo19O+sWogSSRaEL/CST7EDw3LB+ojJHWoG4nGEXHGgV2\n' +
+    'ZTmqHXK/t/XS6PejDrPb2mE20J3Wk+UraeqR9TVDU2bD8q98wo5sPJAleRhgf68jTUTyw2zktd+P\n' +
+    'SvUqWlHmhUjvPPVc1KxJT5ZzSAuhXmj1PNXUs0sa/sqiD63VIFC+vI92cTnOC81M8ncH7abvr2hH\n' +
+    'zZtYsqdcVyBGjUy/t3dgL8TsOlTrT61HI9iZmlVR6xqqBAelweVbNVy1NdsyxHXJtmCUbQkk2/LP\n' +
+    'km3BfiS/vfyH4NkoqOJtaKzX0S/BaERT8gCjTIt2GF57NmG4sqBMdkUvrG/AXmiWMnd7eTtLdPd/\n' +
+    'HmlFY3PHVZX+CmZVMnocdtBehYNz08RYNlu07Io0p/nd4O7uthyXH8m3p2EvhL86VK/D/WlZFdXB\n' +
+    'of3zwYFRhFzrRVuzLUGabRl6yaLM3Fjah2WUR3rc2TaRoGXn0VfBedAi0+PX9B3oHfM+loK542rD\n' +
+    '59H9unJZlYzJcrX187sE59rssNTPdipJg5Umsk9a1ps+LlrWfzJrhc6vL3/Z//jv/+uYJBw25frd\n' +
+    'R81BS1LX5h43CSWdTnJdMiybn/zN3c1P/vbuALSwzIVVR/k0Vdh0u28u28yq8nk0bd/khXIp96st\n' +
+    '0xzUtoDFKTgv0eywEhznOvKx3RjN1qF6Pv1w3nQM3+QZPnn7i8HHP/3yXOfRo2NNBC1Vkw9zT5qY\n' +
+    'rjwb3P/Or/767nkGKcujwcKm29tc1iG1DV4ol3q/2mpZwJLN8xPDgTY7oH19otqkkVFvpqyNRvmg\n' +
+    'XoXPp6dmIPmodzf+JmiRi7u0n1TaRBTYzRVjK8mClEPJ/WMfXfptb3Bpj528lpApbCdQX5NQVsiW\n' +
+    '+oKau1DWVYmtxH611cBxGKfb1f4zJzxkuzQrcxvV0Pe5sE1NpulvGw2QbfdRbx2qHXQLB75Tp0sb\n' +
+    'BS1vf9HTJiJpMjlfVdBSUcnTgdg7SZK8+fHF336HQcpqyGVY+qiuUs8q7ZW5oJoLZdWV2MrtV1vm\n' +
+    'OOhkWXX3MdSLv04G5mXm5FzTre8Mi+4TzQ4sdFOTeYp6Hw0wZU+PjZbFKutQPfZ/YRP4doqslPVr\n' +
+    '0aAl8f4hvO6PUZASJMPNX1383ebHl353E7RS9OQ3TyLOOt76PMHylXaMFbKb0gtldnfva7/q6+hx\n' +
+    'enMV96stc37rcaji/B6n57se72Om+cYbDVhkeRP+skVZRk7f5wALztRhjWTSzDmmAYvvcywr62/o\n' +
+    'sbcNfAsFKxkNWn790y82kyA50aZ+LdrUIw0+28Pkjyc0SPno0sr2R0ksl0Xd5lya0sxV6jsltp+l\n' +
+    'lX1U2klLFme5u3u9g73j+HrZ+9Djck6Pk+nQ60Mt+6FpufM7yyTqOVr2M+WPS3a+93xkU6bJnU99\n' +
+    '2B+XfNk8YZpQsuzAPfg9B2orY5mxfXPH4T2ULev5OvR2idfMzimdh0WPk9MDSUt1GPl+7/lwuL62\n' +
+    'lSA4E6Qza1q9XrIvn+GgPnsILrQ/yl4nwc8P4v72qjfzbGxsRLCcIdW0UZbdro6xLzyrqo9tupD3\n' +
+    'GSKdO0TnFNDvN2asrueSVnj/LssgqwQ9vAerfVWh2ONnipBO2a5Tn8+btj1Gul91IrFbvrMo8l50\n' +
+    '2zZzPux5DJIaZ46FLj9Aehw2CvyZnusx0mOij0m4WWVwMkuujL6F2edSjDR78m9Iy+fehNcqWtZ2\n' +
+    'izQ7tqGeM/snW4oa+CpnZvta1k8hPU6z9kdWh+p59aGP+sZL71YNWvY7a1Gw1rksIURY9HXTYAXp\n' +
+    'gxKLk5WTWLZw7dnh72+wLwq5yhX+TIz0AsZzypEJGEI8WZHF4H5thDnHxx/TsJctbT4mE86lGDyP\n' +
```

Comment:
just noticed, not sure if formatting base64 makes any sense. could have been one liner but doesn't matter :D

Reply:
True that, I simply followed the old wallet formatting as hopefully, this will be thrown away soon with a proper build pipeline.